gocd/alp.target.gocd.yaml

```diff
@@ -0,0 +1,225 @@
+format_version: 3
+pipelines:
+  Installer.Images:
+    group: ALP.Target
+    lock_behavior: unlockWhenFinished
+    materials:
+      repos:
+        git: git://botmaster.suse.de/suse-repos.git
+        auto_update: true
+        whitelist:
+          - SUSE:ALP:Products:Installer:1.0_-_images.yaml
+        destination: repos
+      scripts:
+        auto_update: true
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+        whitelist:
+          - DO_NOT_TRIGGER
+        destination: scripts
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-staging-bot
+    stages:
+    - Expect.Images.To.Finish:
+        resources:
+        - staging-bot
+        tasks:
+        - script: |
+            export PYTHONPATH=scripts
+            set -e
+            ./scripts/gocd/verify-repo-built-successful.py -A https://api.suse.de -p SUSE:ALP:Products:Installer:1.0 -r images
+
+    - Release.Images.To.Test:
+        approval: manual
+        roles:
+        - SLE
+        environment_variables:
+          OSC_CONFIG: /home/go/config/oscrc-totest-manager
+        resources:
+        - staging-bot
+        tasks:
+        - script: |-
+            osc -A https://api.suse.de release SUSE:ALP:Products:Installer:1.0
+            sleep 600
+            while (osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:ToTest/_result?view=summary&repository=images" | grep "result project" | grep -v 'code="published" state="published">'); do
+                echo PENDING
+                sleep 600
+            done
+            osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:ToTest/_result?view=summary&repository=images" | grep "result project" | grep 'code="published" state="published">' && echo PUBLISHED
+
+    - Release.Images.To.Publish:
+        approval: manual
+        roles:
+        - SLE
+        environment_variables:
+          OSC_CONFIG: /home/go/config/oscrc-totest-manager
+        resources:
+        - staging-bot
+        tasks:
+        - script: |-
+            osc -A https://api.suse.de release SUSE:ALP:Products:Installer:1.0:ToTest
+            sleep 600
+            while (osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:PUBLISH/_result?view=summary&repository=images" | grep "result project" | grep -v 'code="published" state="published">'); do
+                echo PENDING
+                sleep 600
+            done
+            osc -A https://api.suse.de/ api "/build/SUSE:ALP:Products:Installer:1.0:PUBLISH/_result?view=summary&repository=images" | grep "result project" | grep 'code="published" state="published">' && echo PUBLISHED
+
+  Installer.Images.NEW:
```

Comment:
indeed, and I'm confused with `Installer.Images`, `Installer.Images.NEW` and `Installer.ImagesManual`, do they really have different meaningful purpose? this looks like just align with what **dolomite.target.gocd.yaml** had though

Reply:
Good point, I would like to remove `Installer.Images.NEW` and `Installer.ImagesManual` (also from SLE and Dolomite) as RelMgr've never used them and also to save agent power on botmaster (that is why I paused  the SLE ones). I will discuss it with RelMgr if they agreed on that. 
Just to give you a background about them:
 - `Installer.Images.NEW`: the only different is that it uses `sle-prjmgr-tools` on `Release.Images.To.Test`, instead of directly run `osc release`. IIRC this script should somehow update the Jira tickets but it is never put in production 
 - `Installer.ImagesManual`: the difference here is that `Release.To.Test` is automatically triggered after `Check.Images.Finish`, this new workflow was requested by RelMgr but never used

I will include `Installer.ReleaseScc` as it was never put in production, as well. For now, I will convert this PR to draft until I have a agreement with RelMgr.